aiida/manage/external/postgres.py

```diff
@@ -166,35 +188,47 @@ def drop_db(self, dbname):
     def copy_db(self, src_db, dest_db, dbuser):
         self.execute(_COPY_DB_COMMAND.format(dest_db, src_db, dbuser))
 
-    def check_db(self, dbname):
-        """Looks up if a database with the name exists, prompts for using or creating a differently named one.
+    def find_new_db(self, start_from='aiida'):
```

Comment:
Should we maybe include `quicksetup` in the label (same goes for the dbuser name) to make it more clear that this was autogenerated? I think this is currently also the case is it not for the database name and user name generated through `verdi quicksetup`

Reply:
When this function is used via quicksetup, it will start from the default user and database name generated by quicksetup, e.g.:
https://github.com/aiidateam/aiida-core/blob/5b6b2b2e2c1d1672e45033dcebf3530626f87bc3/aiida/cmdline/params/options/commands/setup.py#L103

I think this is reasonable, would you agree?

Reply:
I see, the `start_from` will be provided by `verdi quicksetup` itself to contain specific information. If that is the case, then it's all good :+1: